Modules/RobotKit/Sources/Robot+Lights.swift

```diff
@@ -23,6 +23,7 @@ public extension Robot {
         case earRight(in: Color)
 
         case spot(Spot.Position, ids: [UInt8], in: Color)
+        case randomBeltSpots(number: Int, multiColor: Bool = true, in: Color = .black)
```

Comment:
c'est bizarre de pouvoir donner une couleur et aussi pouvoir dire multicolor.

du coup le plus simple serait d'avoir `in colors: [Color]` et tu prends dedans

si tu as besoin de toutes les couleurs, il faut ajouter une conformance à l'enum Color `CaseIterable` pouvoir avoir le `allCases`. il faut surement donner l'implémentation nous même

infos ici: https://www.hackingwithswift.com/example-code/language/how-to-list-all-cases-in-an-enum-using-caseiterable

Reply:
le `allCases` permettra d'avoir `in colors: [Color] = Color.allCases` ou un truc du genre dans la signature